docs/babel.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   presets: ['next/babel', '@zeit/next-typescript/babel'],
   plugins: [
     '@babel/plugin-proposal-optional-chaining',
+    '@babel/plugin-proposal-nullish-coalescing-operator',
```

Comment:
Whice stage is this syntax in?

Reply:
Stage 4 (https://github.com/tc39/proposal-nullish-coalescing)

Reply:
Nice! Sounds better than the `||` operator that ignores falsy values.

Reply:
We should be able to refactor a couple of case in the main repo with it.

Reply:
Yeah, actually used it here to strip the `undefined` or `null` for `boolean ampm`